src/feedback/models/base.py

```diff
@@ -139,8 +139,10 @@ class FachgebietEmail(models.Model):
 
     @staticmethod
     def assign_to_fachgebiet(email):
-        fachgebiet_id = FachgebietEmail.objects.get(email_suffix__contains=email).fachgebiet_id
-        return Fachgebiet.objects.get(pk=fachgebiet_id)
+        try:
+            return Fachgebiet.objects.get(pk=FachgebietEmail.objects.get(email_suffix__contains=email).fachgebiet_id)
+        except Exception:
+            pass
```

Comment:
I already mentioned it in a later commit but this is not a fix. There is the .exists() method to check if a queryset is empty or not.